lib/hooks/useCommonData/useCommonData.js

```diff
@@ -51,7 +52,7 @@ const useCommonData = (tenantId) => {
     const loadedData = {};
 
     if (isConsortiaEnv(stripes)) {
-      loadedData.locations = locationsFromAllTenants;
+      loadedData.locations = uniqBy([...locationsFromAllTenants, ...locations], 'id');
```

Comment:
Won't the locations from current tenant be included in `locationsFromAllTenants?` Why do we need to add them separately?

Reply:
For some reason in `locationsFromAllTenants` there are not full list of locations. That's why I decided to merge `locationsFromAllTenants` and `locations`. `locations` variable contains missing locations.

Reply:
maybe there is some issue on the BE side as to why `locationsFromAllTenants` doesn't contain all locations, or in `useLocationsFromAllTenantsQuery` query. Let's investigate it more, it looks like an unnecessary additional operation or quick workaround.

Reply:
@mariia-aloshyna @BogdanDenis All tenant's locations become available after reindexing the locations. Closing this PR.